Gemfile

```diff
@@ -27,6 +27,6 @@ else
   gem 'puppet', :require => false
 end
 
-gem 'hocon', '~> 0.9.3',       :require => false
```

Comment:
Would it make sense to pin this to like 1.x, or 1.2.x or something?  I'm not up to speed on what our best practices are around versioning gem dependencies for modules, so I'm OK with not specifying a version range at all if that's the normal modus operandi for modules, but we've been bitten a few times in the past by the lack of determinism that results from not restricting version ranges, so I figured it was worth asking.